holland/legacy/config.py

```diff
@@ -25,8 +25,14 @@ def extract_params(section):
 def get_config(**kwargs):
     global config
     global spec
+
+    holland_conf = '/etc/holland/holland.conf'
+    currplatform = sys.platform
```

Comment:
Typos on my part. I can fix it soon if needed, otherwise it does work as expected. I could even better yet just drop the variable and use sys.platform directly. Up to @abg and how he prefers the code be written I guess, at least as far as the PR is concerned.